addons/l10n_ec/data/template/account.tax-ec.csv

```diff
@@ -1,349 +1,349 @@
 "id","name","type_tax_use","amount_type","sequence","amount","description","invoice_label","l10n_ec_code_base","l10n_ec_code_applied","tax_group_id","l10n_ec_code_ats","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","name@es_EC"
-"tax_vat_411","VAT 12%","sale","percent","9","12.0","","VAT 12%","411","421","tax_group_vat_12","","base","invoice","+411 (Reporte 104)","","","Iva 12%"
+"tax_vat_411","12%","sale","percent","9","12.0","VAT 12%","12%","411","421","tax_group_vat_12","","base","invoice","+411 (Reporte 104)","","","Iva 12%"
 "","","","","","","","","","","","","tax","invoice","+421 (Reporte 104)","ec_sale_vat","",""
 "","","","","","","","","","","","","base","refund","-411 (Reporte 104)","","",""
 "","","","","","","","","","","","","tax","refund","-421 (Reporte 104)","ec_sale_vat","",""
-"tax_vat_412","VAT 12% (assets)","sale","percent","10","12.0","","VAT 12%","412","422","tax_group_vat_12","","base","invoice","+412 (Reporte 104)","","","Iva 12% (activos)"
+"tax_vat_412","12% A","sale","percent","10","12.0","VAT 12% (assets)","12%","412","422","tax_group_vat_12","","base","invoice","+412 (Reporte 104)","","","Iva 12% (activos)"

```

Comment:
We should make sure how it looks in the invoice subtotals at the bottom of the invoice, if I am not mistaken the description used to be shown there.